.doc_gen/metadata/forecast_metadata.yaml

```diff
@@ -0,0 +1,183 @@
+# zexi 0.4.0
+forecast_CreateDataset:
+  title: Create a data set for the &FORlong; by using an &AWS; SDK
```

Comment:
Other examples don't have "by" in these, is this consistent with other services? For example, I see "Create an AWS Glue Crawler using an AWS SDK". Same question for all of the actions.